src/Test/XAML/FeatureTests/XamlTests35/Microsoft/Test/Xaml/Parser/MethodTests/XmlContent/XmlHandlingTest.cs

```diff
@@ -210,34 +211,59 @@ public void VerifyXmlAttributeProperties()
             // GetXmlNamespaceMaps(DependencyObject dependencyObject)
             // SetXmlNamespaceMaps(DependencyObject dependencyObject, string value)
             //
-            // NOTE: These methods are broken and won't be fixed.  We're just calling
-            // them to keep them off the radar.  The exception verification will
-            // alert us if the implementation is ever fixed.
-            //
-            Exception ex = null;
+
             try
             {
-                XmlAttributeProperties.SetXmlNamespaceMaps(dobj, "foo");
+                //Pass null to both dependencyObject and value in SetXmlNamespaceMaps, this will throw exception
+                XmlAttributeProperties.SetXmlNamespaceMaps(null, null);
```

Comment:
Here we are testing for null value just to see if it results in exception or not because `SetXmlNamespaceMaps ` will throw exception in that scenario.
However for the correct value test we are checking that as well, please refer the code snippet from line 237 to 262.